src/library.js

```diff
@@ -998,17 +998,24 @@ LibraryManager.library = {
     if (_tzset.called) return;
     _tzset.called = true;
 
+    var currentYear = new Date().getFullYear();
+    var winter = new Date(currentYear, 0, 1);
+    var summer = new Date(currentYear, 6, 1);
+    var winterOffset = winter.getTimezoneOffset();
+    var summerOffset = summer.getTimezoneOffset();
+
+    function stdTimezoneOffset() {
```

Comment:
I don't know much about timezones, sorry. Is it obvious what this means? Please add a  comment explaining the name of the function + why the max of the winter and summer offsets is the value we need (how it relates to UTC etc.).

Reply:
Added a comment which hopefully clears this up, the comment below also goes into some detail as to why we need it and links to the pubs page.  I also merged with master so hopefully that fixes the CI issues. Thanks.